config/config.go

```diff
@@ -23,10 +23,19 @@ import (
 	_ "embed"
 	"fmt"
 	"os"
+	"path/filepath"
 
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	DEFAULT_SYS_CFG_PATH    = "config/systems.yaml"
+	DEFAULT_REPO_CFG_PATH   = "config/repositories.yaml"
+	DEFAULT_DAEMON_CFG_PATH = "config/daemons.yaml"
+)
+
+var _configPath string
```

Comment:
That underscore doesn't feel go-like.

You just need to have the variable begin with a lower-case letter and it'll be scoped to the package, which in this case is the 'config' package.

Likewise, by beginning your constants (DEFAULT_*) with uppercase letters you're allowing them to be visible outside the 'config' package.  But I see you want that for main.go.  Also, Go prefers mixed caps.   So a more go-like way is to name them like "DefaultSysCfgPath".

Look up "underscore" in https://go.dev/doc/effective_go.

Reply:
I originally had a local variable that was setting `_configPath` so I needed a way to set it apart. I chaged it to this and I completely forgot to come back to this to give it a better name. My bad.

Is there a convention for competing global/local variable names? Maybe just don't do it?